Makefile

```diff
@@ -0,0 +1,5 @@
+
+.PHONY: install
+install:
+	# TODO use vault '--ask-vault-pass'
+	ansible-playbook -i '127.0.0.1,' --ask-become-pass linux.yaml
```

Comment:
Is `-i` necessary if you've declared `hosts: localhost` in `linux.yaml`?

Reply:
I am not sure, I think so.

the hosts list in linux.yaml affects which hosts are covered by that YAML file, but the flag is to specify which ones are being acted on with that command.